### ISSUE TITLE
"Could you provide the complete `requirements.txt` file?

### ISSUE THREAD

**Issue by Zee-hash** [cnn-OvO/pulsar-identification#1]
I seem to have encountered issues with the versions of certain libraries like scipy, numpy, pandas, etc. when testing the model you provided. Here is the error message:：
```
find 493 pfd in 0.00
start.
Traceback (most recent call last):
  File "../pred.py", line 156, in <module>
    pred.topmodel_pred(CurDir+"/trained_model/top_lr0.1_newtrain1.model",   # model for Top LR layer
  File "../pred.py", line 122, in topmodel_pred
    X1 = self.profdm_ann_test(modelname=model1,num_works=num_works)
  File "../pred.py", line 79, in profdm_ann_test
    clf = pickle.load(f)
_pickle.UnpicklingError: invalid load key, 'v'.
```
The versions of the libraries I currently have installed are as follows.
```
Package           Version
----------------- ------------
einops            0.4.1
numpy             1.22.4
opencv-python     3.4.10.37
pandas            1.4.0
Pillow            10.1.0
pip               23.3
python-dateutil   2.8.2
pytz              2023.3.post1
scipy             1.8.1
setuptools        68.0.0
six               1.16.0
torch             1.10.0
torchvision       0.11.1
typing_extensions 4.8.0
wheel             0.41.2
```

**Comment by cnn-OvO** [comment]
Maybe you can install 'pickle' of version 0.7.5

**Comment by Zee-hash** [comment]
> Maybe you can install 'pickle' of version 0.7.5

First of all, thank you very much for your reply. However, it seems that I cannot select a version for the 'pickle' module, as it is a built-in module in Python. Could you please provide more details about which module you are referring to?

**Comment by Zee-hash** [comment]
"I think I've found the issue. I checked the contents of the 'trained_model' folder and discovered that it was caused by Git Large File System (Git LFS). Since I hadn't used LFS before, when I used `git clone`, the 'trained_model' content wasn't fully downloaded, but only a small pointer of less than 1kB. The problem was resolved after I used:

```markdown
apt install git-lfs
git lfs install
git lfs pull
```

Thank you very much for your help. I will close this issue. Have a wonderful day!"
